tests/voq/files/CRC_Error_LC

```diff
@@ -0,0 +1,10 @@
+"port enable eth yes"
```

Comment:
We shouldn't have CINT code or bcm calls here. Please see if this can be achieved via SAI API.

Reply:
Hello @vmittal-msft ,
At the moment, there is no API available to accomplish this task. We can proceed with the enhancement once Broadcom introduces support for it.